geojson/validation.py

```diff
@@ -0,0 +1,97 @@
+import geojson
+
+
+def IsValid(obj):
```

Comment:
What do you think about renaming this to `is_valid`? Then it will follow the PEP8 naming conventions

https://www.python.org/dev/peps/pep-0008/#naming-conventions